src/class.js

```diff
@@ -18,7 +25,24 @@
 // Cat should have the property `name` that is set in the constructor and the method
 // `meow` that should return the string `<name> meowed!` where `<name>` is the `name`
 // property set on the Cat instance.
+class Animal {
+  constructor(options) {
+    this.age = options.age;
+  }
+  growOlder() {
+    return this.age++;
```

Comment:
You'll probably need to move the `++` to the front like this: `++this.age;`.  That makes it so the number is incremented BEFORE it is returned

Reply:
postfix operators are kind of funny like that.